src/components/table/LogTable.vue

```diff
@@ -35,10 +35,17 @@
 					</td>
 				</tr>
 
-				<LogTableRow v-for="row, rowNumber in sortedRows"
-					:key="rowNumber"
-					:row="row"
-					@show-details="showDetailsForRow" />
+				<template v-for="(row, rowNumber) in sortedRows">
+					<tr v-if="checkIfDummy(rowNumber)" :key="rowNumber" class="dummy">
+						<td colspan="5" class="log-table__load-more">
+							{{ t('logreader', 'Loading older log entries') }}
+						</td>
+					</tr>
```

Comment:
Should be even more performant to render empty space with no content
```suggestion
					<tr v-if="checkIfDummy(rowNumber)" :key="rowNumber" class="dummy" />
```